CONTRIBUTORS.rst

```diff
@@ -0,0 +1,10 @@
+Contributors
+============
+
+* Alexis Metaireau <alexis@mozilla.com>
+* Fil <fil@rezo.net>
+* Mathieu Agopian <mathieu@agopian.info>
+* Mathieu Leplatre <mathieu@mozilla.com>
+* Nicolas Perriault <nperriault@mozilla.com>
+* Rémy Hubscher <rhubscher@mozilla.com>
+* Tarek Ziade <tarek@mozilla.com>
```

Comment:
I have updated this: https://github.com/mozilla-services/kinto/pull/86#issue-91756118

Not sure if we put the template somewhere else.